_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
+| {% include anchor a="error.consumption.attributes.predecessorIsNotOwnSharedRelationshipAttribute" %}            | The predecessor is not an [own shared RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                             |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotPeerSharedIdentityAttribute" %}               | The predecessor is not a [peer shared IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                                 |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotPeerSharedRelationshipAttribute" %}           | The predecessor is not a [peer shared RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                             |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotRepositoryAttribute" %}                       | The predecessor is not a [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                                           |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotThirdPartyOwnedRelationshipAttribute" %}      | The predecessor is not a [third party owned RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                       |
+| {% include anchor a="error.consumption.attributes.predecessorSourceAttributeIsNotRepositoryAttribute" %}        | The predecessor source Attribute is not a [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                          |
+| {% include anchor a="error.consumption.attributes.predecessorSourceContentIsNotEqualToCopyContent" %}           | The predecessor source Attribute `content` doesn't match the content of the predecessor shared Attribute copy. ([Update Attributes by succession]({% link _docs_integrate/update-attributes-by-succession.md %}))                                                                                           |
```

Comment:
"`content`" instead of "content" (I am referring to the second occurrence of the word).

Also, use a link to the LocalAttribute here (and in other descriptions, where Attributes are mentioned).